src/main/java/com/whereisagift/wishlist/Wishlist.java

```diff
@@ -0,0 +1,26 @@
+package com.whereisagift.wishlist;
+
+import com.whereisagift.user.User;
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@Table(name = "wishlists")
+@NoArgsConstructor
+public class Wishlist {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "name")
+    private String name;
+
+    @ManyToOne
+    @JoinColumn(name = "creator_id")
+    private User user;
```

Comment:
```suggestion
    private User creator;
```

Так будет корректнее